src/style.css

```
@@ -1 +1,82 @@
 /* Styles go here */
+@import url(https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap);
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
+}
+
+.logo {
+  display: flex;
+  align-self: center;
+  width: 40px;
+  height: 40px;
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+  height: 60px;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
+  align-items: center;
+  padding-left: 0;
+}
+
+.list__item {
+  padding-left: 20px;
+}
+
+.nav__list li:first-child {
+  padding-left: 0;
+}
+
+.nav__link {
+  text-decoration: none;
+  color: #000;
+  padding: 23px 0;
+  line-height: 14px;
+  cursor: pointer;
+  transition: all 0.3s ease-in-out;
```

Don't use all in transition, set properties explicitly you 
```suggestion
  transition: color 0.3s ease-in-out;
```